website/docs/docs/deploy/deploy-environments.md

```diff
@@ -186,6 +190,37 @@ This section allows you to determine the credentials that should be used when co
 </WHCode>
 
 
+## Staging environment <Lifecycle status='beta' />
+
+:::note
+Currently in limited availability beta.
+:::
+
+Staging environments are a useful way to grant developers access to deployment workflows and tools while limiting access to production data. Staging environments are configured with their own, long living branch (for example, `staging`) that may be very similar to `main` in many ways, but limits the data that the developers can access. 
+
+Ideally, the workflows would move upstream from Developer environment -> Staging environment -> Production environment with developer branches feeding in to the staging branch, then ultimately `main`. In many cases, `main` and `staging` branches will be identical after a merge, and remain that way until the next batch of changes from `developer` branches are ready to be elevated. We recommend setting branch protection rules on `stagin` similar to `main`.
+
+### Create a staging environment 
+
+In the dbt Cloud, navigate to **Deploy** -> **Environments** and then click **Create Environment**. Select **Deployment** as the environment type. If you already have a development environment, the option will be greyed out.
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/create-staging-environment.png" width="85%" title="Create a staging environment" />
+
+The "dbt version" should match the version used in production. 
```

Comment:
A year ago, I might have said: you could use a Staging environment to try out a newer dbt version

Now I'd say: given that we're going to be encouraging everyone to "Keep on latest version" very soon, I don't think we need this sentence here